index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mi CV</title>
+</head>
+<body>
+    <!-- Aquí va el contenido de tu CV -->
+     <header>
+     <h1>Gabriela Bejarano Pérez</h1>
+     <h2>Información Personal</h2>
+     <ul>
+        <li>gabrielabejarano09@gmail.com</li>
+        <li>314 2425931</li>
+        <li>Diagonal 4b bis 29 23</li>
+     </ul>
+    </header>
+    <main>
+    <h2>Educación</h2>
+     <h3>Bachiller</h3>
+     <p> Acellus Academy | 2020-2023, Kansas City, Missouri</p>
+     <h3>Programacion y Tecnologias Digitales</h3>
+     <p> Universidad Tecnologia de Pereira | Ministerio de las Tecnologias | 2022, Bogotá, Colombia </p>
+     <h3>Ingenieria Informática</h3>
+     <p> Universidad de la Sabana | 2023-Cursando Actualmente | Tercer Semestre, Chía, Colombia</p>
+     <h2>Experiencia Laboral</h2>
+     <h3>Proyecto Biblioteca en Java</h3>
+     <p>2023</p>
+     <p>Desarrollé un sistema de gestión para una biblioteca utilizando Java, enfocado en la administración eficiente de bibliotecarios, 
+        clientes y libros. El proyecto incluyó:
+     </p>
+     <ul>
+        <li>Desarrollo de Software: Creación de un programa en Java para el control y gestión de bibliotecas. </li>
+        <li>Interfaz Gráfica: Implementación de una interfaz gráfica de usuario intuitiva para facilitar la interacción del usuario final.</li>
+        <li>Gestión de Usuarios: Control de acceso y administración tanto para bibliotecarios como para clientes, con funcionalidades específicas para cada rol.</li>
+        <li>Manejo de Libros: Registro, búsqueda y gestión de libros en el sistema, incluyendo características como disponibilidad y préstamo.</li>
+     </ul>
+     <h2>Habilidades</h2>
+     <p> Java, Python, HTML, CSS, Creatividad, Trabajo Colaborativo, Resolución de Problemas</p>
```

Comment:
```suggestion
<ul>
  <li>Java</li>
  <li>Python</li>
  <li>HTML</li>
  <li>CSS</li>
  <li>Creatividad</li>
  <li>Trabajo Colaborativo</li>
  <li>Resolución de Problemas</li>
</ul>
```

Al ser una lista, es más semántico representarlo así